### ISSUE TITLE
✅ 🔗 🐞 Bug Report: Setting top level container to HTML Tag a (link) breaks the layout of Containers inside the Loop Carousel (Duplicate of #23087) - [ED-11293]

### ISSUE THREAD

**Issue by Grassmedia** [elementor/elementor#23103]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Loop Carousel - Setting top-level container of the loop template to HTML Tag a (link) breaks the layout

### Steps to reproduce

Create a new loop item (same for products & posts).  Set the top-level container to HTML tag a (link).

**Without a link in loop **
![without a link](https://github.com/elementor/elementor/assets/57148940/e51a3fcc-7f30-481f-9f50-9b8842c001b9)

**With a link in loop**
![with a link](https://github.com/elementor/elementor/assets/57148940/cc8a5aac-3616-4602-a145-d07c7e07ad55)

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt



== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.4.20
	PHP Version: 8.0.29
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://ccm2023.myclientsites.co.uk
	Home URL: https://ccm2023.myclientsites.co.uk
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Activity Log
		Version: 2.8.6
		Author: Activity Log Team

	Advanced Order Export For WooCommerce
		Version: 3.4.0
		Author: AlgolPlus

	Code Snippets
		Version: 3.4.2
		Author: Code Snippets Pro

	Complianz Privacy Suite (GDPR/CCPA) premium
		Version: 6.3.6
		Author: Really Simple Plugins

	Elementor
		Version: 3.14.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.1
		Author: Elementor.com

	Flexible Checkout Fields
		Version: 3.5.10
		Author: WP Desk

	FluentSMTP
		Version: 2.2.5
		Author: FluentSMTP & WPManageNinja Team

	Imagify
		Version: 2.1.1
		Author: Imagify – Optimize Images & Convert WebP

	JetEngine
		Version: 3.2.2
		Author: Crocoblock

	JetSmartFilters
		Version: 3.1.2
		Author: Crocoblock

	JetWooBuilder For Elementor
		Version: 2.1.5
		Author: Crocoblock

	MainWP Child
		Version: 4.4.1.3
		Author: MainWP

	Rank Math SEO
		Version: 1.0.119.1
		Author: Rank Math

	Regenerate Thumbnails
		Version: 3.1.5
		Author: Alex Mills (Viper007Bond)

	Sequential Order Numbers for WooCommerce
		Version: 1.9.7
		Author: SkyVerge

	Site Kit by Google
		Version: 1.105.0
		Author: Google

	WooCommerce
		Version: 7.9.0
		Author: Automattic

	Woocommerce Booking Exporter
		Version: 1.6.8
		Author: wpexperts.io

	WooCommerce Bookings
		Version: 1.16.02
		Author: WooCommerce

	WooCommerce Cart Abandonment Recovery
		Version: 1.2.25
		Author: CartFlows Inc

	WooCommerce Force Sells
		Version: 1.2.0
		Author: Themesquad

	WooCommerce Stripe Gateway
		Version: 7.4.1
		Author: WooCommerce

	WP All Export - User Export Add-On Pro
		Version: 1.0.7
		Author: Soflyy

	WP All Export Pro
		Version: 1.8.0
		Author: Soflyy

	WP File Manager
		Version: 7.1.9
		Author: mndpsingh287

	WP Reset
		Version: 1.97
		Author: WebFactory Ltd

	WP Rocket
		Version: 3.14.1
		Author: WP Media

	WP Social Ninja
		Version: 3.9.4
		Author: WPManageNinja LLC

	WP Social Ninja Pro
		Version: 3.9.1
		Author: WPManageNinja LLC


== Must-Use Plugins ==
	Object Cache Pro (MU)
		Version: 1.18.2
		Author: Rhubarb Group


== Elements Usage ==
	
	footer : 1
		container : 4
		image : 1
		nav-menu : 1
		text-editor : 2
	header : 3
		container : 3
		global : 2
		heading : 4
		icon : 3
		icon-list : 1
		image : 1
		nav-menu : 1
		woocommerce-menu-cart : 2
	page : 2
		button : 1
		global : 1
		heading : 6
		text-editor : 1
	section : 18
		button : 1
		countdown : 2
		form : 2
		global : 3
		google_maps : 2
		heading : 11
		image : 6
		nav-menu : 3
		share-buttons : 3
		sitemap : 1
		social-icons : 1
		text-editor : 5
		woocommerce-menu-cart : 1
	single : 3
		form : 1
		google_maps : 1
		heading : 4
		image : 3
		nav-menu : 1
		social-icons : 1
		text-editor : 2
	widget : 0
		countdown : 1
		form : 1
		google_maps : 1
		nav-menu : 1
		toggle : 1
	wp-post : 0
		container : 10
		countdown : 1
		form : 2
		global : 2
		google_maps : 2
		heading : 22
		image : 7
		loop-carousel : 1
		nav-menu : 2
		shortcode : 1
		template : 1
		text-editor : 19
		wc-elements : 4
		woocommerce-menu-cart : 1
	wp-page : 17
		container : 1
		global : 1
		heading : 7
		image : 1
		loop-grid : 1
		shortcode : 3
		text-editor : 3
		wp-social-ninja-instagram-widget : 1
	not-supported : 0
		heading : 4
		text-editor : 9
	loop-item : 4
		container : 3
		heading : 2
		text-editor : 1
		woocommerce-product-price : 1
		woocommerce-product-title : 1
	jet-woo-builder-shop : 0
		jet-woo-products : 1
	jet-woo-builder-checkout : 0
		heading : 1
		jet-checkout-additional-form : 1
		jet-checkout-billing : 1
		jet-checkout-order-review : 1
		jet-checkout-payment : 1
		text-editor : 1
	jet-woo-builder-cart : 0
		heading : 2
		jet-cart-empty-message : 1
		jet-cart-return-to-shop : 2
		jet-cart-table : 1
		jet-cart-totals : 1
	jet-woo-builder : 5
		heading : 3
		image : 1
		jet-single-add-to-cart : 1
		jet-single-excerpt : 1
		jet-single-price : 1
		text-editor : 2
	jet-listing-items : 0
		heading : 1
		jet-listing-dynamic-field : 1
		jet-listing-dynamic-image : 1
		jet-listing-dynamic-link : 1


== Settings ==
	
	cpt_support: post, page, jet-woo-builder, jet-woo-builder
	allow_tracking: yes
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active
	woocommerce: Active


== Elementor Experiments ==
	Optimised DOM Output: Inactive by default
	Improved asset loading: Inactive by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Inactive
	Hello Theme Header &amp; Footer: Inactive by default
	Editor Top Bar: Inactive by default
	Landing pages: Inactive
	Nested Elements: Inactive by default
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 322023-04-06 16:15:48 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-06 16:15:48 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-06 16:15:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.12.1',
)]
2023-04-06 16:15:52 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.12.1',
)]
2023-04-13 07:44:20 [info] elementor-pro::elementor_pro_updater Started 
2023-04-13 07:44:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-13 07:44:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-13 07:44:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-07-17 08:52:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.14.1',
)]
2023-07-17 08:52:54 [info] elementor-pro::elementor_pro_updater Started 
2023-07-17 08:52:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-07-17 08:52:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-07-17 08:52:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.14.1',
)]
2023-07-17 08:52:56 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.14.1',
)]
2023-07-17 08:55:12 [info] elementor::elementor_updater Started 
2023-07-17 08:55:12 [info] Elementor/Upgrades - _on_each_version Start  
2023-07-17 08:55:12 [info] Elementor/Upgrades - _on_each_version Finished 
2023-07-17 08:55:12 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2023-07-17 08:55:12 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-07-17 08:55:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.14.1',
)]

PHP: showing 9 of 9PHP: 2023-05-07 20:36:25 [warning X 2][/home/277156.cloudwaysapps.com/xkbejsfepc/public_html/wp-content/plugins/elementor-pro/core/utils.php::228] Attempt to read property &quot;post_author&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 08:04:29 [warning X 1][/home/277156.cloudwaysapps.com/xkbejsfepc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 08:52:21 [warning X 15][/home/277156.cloudwaysapps.com/xkbejsfepc/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::734] Attempt to read property &quot;label&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 11:25:57 [warning X 11][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 11:25:57 [warning X 19][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-07-18 11:30:47 [error X 1][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions()
#2 ../wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 ../wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request()
#4 ../wp-includes/class-wp-hook.php(332): WP_Hook-&gt;apply_filters()
#5 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action()
#6 ../wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 12:11:38 [warning X 4][../wp-content/plugins/elementor/includes/base/controls-stack.php::1432] Undefined array key &quot;_element_width_tablet_extra&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 12:16:38 [warning X 2][../wp-content/plugins/elementor/includes/base/controls-stack.php::1432] Undefined array key &quot;columns_tablet_extra&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-18 14:48:21 [warning X 2][../wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-07-18 14:30:02 [error X 5][https://ccm2023.myclientsites.co.uk/wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.14.1:3:107466] Routes: `panel/editor/settings` not found.



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	JetWooBuilder For Elementor: Compatible

== Elementor Pro - Compatibility Tag ==
	
	JetWooBuilder For Elementor: Compatible


````

</details>

**Comment by 79ho3ein** [comment]
It seems that you are using the Loop carousel widget. Yes, this widget has this bug and it has already been reported. But ...

**Comment by Grassmedia** [comment]
Yep, quite right!  How did I miss that warning!

**Comment by nicholaszein** [comment]
Hey Nigel!

Thank you for your report! 🙏 

Although your issue might be because you added another element with an anchor link inside as @m1ga pointed out, @79ho3ein is correct to point out that it seems there is an issue related to some Swiper styles that are overriding the display property of Containers with `<a>` tags. More specifically the following declaration:

```css

.swiper-slide a {
    display: inline;
}

```

We are aware of this issue and are discussing a solution.

Meanwhile, you can use the workaround in [this comment](https://github.com/elementor/elementor/issues/23087#issuecomment-1657000548). 😉 

### 🔗 🐞 This issue is a Duplicate of #23087

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

Kind regards

**Comment by nicholaszein** [comment]
Hey guys!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
